frontend/src/app/organization/organization-contact/organization-contact.component.ts

```diff
@@ -0,0 +1,30 @@
+import {Component, ElementRef, EventEmitter, Input, OnChanges, Output, SimpleChanges, ViewChild} from '@angular/core';
```

Comment:
make this import consistent with others (AFAIK all import have space just after `{` and before `}`

Reply:
AFAIK  you're totally right. It was about time to change IDE settings to prevent it from removing those spaces every time when import statement was added automatically.